modules/demo/spatial/util.js

```diff
@@ -141,13 +141,15 @@ export async function loadPoints() {
   async function loadPointsTable(loadDatasetIfNotFound = true) {
     try {
       console.time(`load points Arrow table (${(168898952).toLocaleString()} points)`);
-      const table =
-        await Table.from(createReadStream(Path.join(__dirname, 'data', '168898952_points.arrow')));
+      const filePath = Path.join(__dirname, 'data', '168898952_points.arrow');
+      if (!existsSync(filePath)) { throw new Error('file not found'); }
```

Comment:
Is this necessary? I think the `await Table.from(...)` line below this should throw if the file doesn't exist.

Reply:
exactly what I thought, but when I tested it locally, it wasn't throwing any error, and the function was returning an empty arrow table, whenever a file was not found